backend/zeno_backend/server.py

```diff
@@ -175,14 +175,19 @@ def get_slices(project: str, request: Request):
         return select.slices(project)
 
     @api_app.get(
-        "/charts/{project}",
+        "/charts/{owner}/{project}",
         response_model=list[Chart],
         tags=["zeno"],
     )
-    def get_charts(project: str, request: Request):
-        if not util.access_valid(project, request):
+    def get_charts(owner_name: str, project_name: str, request: Request):
+        project_uuid = select.project_uuid(owner_name, project_name)
```

Comment:
Can't we do this from the project uuid? Should have this already when fetching charts, no?